phys/module_radiation_driver.F

```diff
@@ -5073,97 +5076,133 @@ SUBROUTINE gt_aod(p_phy,DZ8W,t_phy,qvapor, nwfa,nifa, taod5503d,  &
       lookup_tabl(3,3,1) = 7.23108E-15  
       lookup_tabl(3,3,2) = 2.77201E-12
+      lookup_tabl(3,3,3) = 1.11447E-15
       lookup_tabl(3,4,1) = 7.21800E-15  
       lookup_tabl(3,4,2) = 2.77111E-12
+      lookup_tabl(3,4,3) = 1.11061E-15
 
       lookup_tabl(4,1,1) = 8.95130E-15  
       lookup_tabl(4,1,2) = 2.87263E-12
+      lookup_tabl(4,1,3) = 1.36116E-15
       lookup_tabl(4,2,1) = 9.01582E-15  
       lookup_tabl(4,2,2) = 2.87252E-12
+      lookup_tabl(4,2,3) = 1.35479E-15
       lookup_tabl(4,3,1) = 9.13216E-15  
       lookup_tabl(4,3,2) = 2.87241E-12
+      lookup_tabl(4,3,3) = 1.34787E-15
       lookup_tabl(4,4,1) = 9.16219E-15  
       lookup_tabl(4,4,2) = 2.87211E-12
+      lookup_tabl(4,4,3) = 1.33910E-15
 
       lookup_tabl(5,1,1) = 1.06695E-14  
       lookup_tabl(5,1,2) = 2.96752E-12
+      lookup_tabl(5,1,3) = 1.58848E-15
       lookup_tabl(5,2,1) = 1.06370E-14  
       lookup_tabl(5,2,2) = 2.96726E-12
+      lookup_tabl(5,2,3) = 1.57854E-15
       lookup_tabl(5,3,1) = 1.05999E-14  
       lookup_tabl(5,3,2) = 2.96702E-12
+      lookup_tabl(5,3,3) = 1.56648E-15
       lookup_tabl(5,4,1) = 1.05443E-14  
       lookup_tabl(5,4,2) = 2.96603E-12
+      lookup_tabl(5,4,3) = 1.55057E-15
 
       lookup_tabl(6,1,1) = 1.37908E-14  
       lookup_tabl(6,1,2) = 3.15081E-12
+      lookup_tabl(6,1,3) = 2.02033E-15
       lookup_tabl(6,2,1) = 1.37172E-14  
       lookup_tabl(6,2,2) = 3.15020E-12
+      lookup_tabl(6,2,3) = 1.99948E-15
       lookup_tabl(6,3,1) = 1.36362E-14  
       lookup_tabl(6,3,2) = 3.14927E-12
+      lookup_tabl(6,3,3) = 1.97488E-15
       lookup_tabl(6,4,1) = 1.35287E-14  
       lookup_tabl(6,4,2) = 3.14817E-12
+      lookup_tabl(6,4,3) = 1.94523E-15
 
       lookup_tabl(7,1,1) = 2.26019E-14  
       lookup_tabl(7,1,2) = 3.66798E-12
+      lookup_tabl(7,1,3) = 3.14156E-15
       lookup_tabl(7,2,1) = 2.24435E-14  
       lookup_tabl(7,2,2) = 3.66540E-12
+      lookup_tabl(7,2,3) = 3.08114E-15
       lookup_tabl(7,3,1) = 2.23254E-14  
       lookup_tabl(7,3,2) = 3.66173E-12
+      lookup_tabl(7,3,3) = 3.01021E-15
       lookup_tabl(7,4,1) = 2.20496E-14  
       lookup_tabl(7,4,2) = 3.65796E-12
+      lookup_tabl(7,4,3) = 2.92456E-15
 
       lookup_tabl(8,1,1) = 4.41983E-13  
       lookup_tabl(8,1,2) = 7.50091E-11
+      lookup_tabl(8,1,3) = 1.95503E-14
       lookup_tabl(8,2,1) = 3.93335E-13  
       lookup_tabl(8,2,2) = 6.79097E-11
+      lookup_tabl(8,2,3) = 1.74308E-14
       lookup_tabl(8,3,1) = 3.45569E-13  
       lookup_tabl(8,3,2) = 6.07845E-11
+      lookup_tabl(8,3,3) = 1.53194E-14
       lookup_tabl(8,4,1) = 2.96971E-13  
-      lookup_tabl(8,4,2) = 5.36085E-11     
+      lookup_tabl(8,4,2) = 5.36085E-11
+      lookup_tabl(8,4,3) = 1.32479E-14     
 
       DO j=jts,jte
          DO k=kts,kte
             DO i=its,ite
                AOD_wfa(i,k,j) = 0.
                AOD_ifa(i,k,j) = 0.
+               if ( wif_input_opt .eq. 2 ) then
+                  AOD_bca(i,k,j) = 0.
+               end if
```

Comment:
@twjuliano 
Tim,
For performance (I know, it is just 1 3d array), would you make two DO loops. The second one has the outer `if ( wif_input_opt .eq. 2 ) then`, and then the inned nest DO loop.

Reply:
Ah, screw it, they are all over the place. Never mind